l2-contracts/contracts/ConsensusRegistry.sol

```diff
@@ -0,0 +1,278 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.20;
+
+import {EfficientCall} from "@matterlabs/zksync-contracts/l2/system-contracts/libraries/EfficientCall.sol";
+
+/// @title ConsensusRegistry
+/// @dev A contract to manage consensus nodes and committees.
+contract ConsensusRegistry {
+    // Owner of the contract (i.e., the governance contract).
+    address public owner;
```

Comment:
The contract doesn't have the functionality to transfer ownership. This is basic functionality that is better to have not to redeploy the contract if the Abstract Governance protocol changes.

Reply:
Please use import and use `Ownable2Step` from OZ library instead. 
- https://github.com/matter-labs/era-contracts/blob/vb-add-admin-contracts/l1-contracts/contracts/governance/ChainAdmin.sol#L5

Reply:
Added here: https://github.com/matter-labs/era-contracts/pull/555/commits/08b8dbf8922f64f5ee8a213b3dea7ebd0650e26a